src/DS/rds.c

```diff
@@ -601,6 +602,13 @@ static void ds_rds_add_xccdf_test_results(xmlDocPtr doc, xmlNodePtr reports,
 {
 	xmlNodePtr root_element = xmlDocGetRootElement(xccdf_result_file_doc);
 
+	if (root_element->ns && root_element->ns->href &&
```

Comment:
I'd have to change the entire function for that to have any effect. The rest of the function is unrelated to this change. Furthermore, the function is only used internally.

Do you feel strongly about adding the check?